test/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig",
   "baseUrl": ".",
-  "include": ["**/*.ts"],
+  "include": ["**/*.ts", "./api/utils/constants.ts"],
```

Comment:
You can remove this change - sorry...this was something I was trying to do to get the imported freshtrak urls to not be undefined

Reply:
I also added  this line in locationService.ts locally and tested it.

```
const response = await makeSCOSRequest<ScosAgencyDto[]>(query);
```
Is this what you were thinking of?  I so, I will add it to the next (and hopefully last) commit for this PR.

Reply:
Yup exactly!